### PR TITLE
Configured Postgresql for backend

### PR DESCRIPTION
**Pull request by djsbalakrishnan**
Following files were changed 
 1. requirements.txt
 2. settings.py
 3. models.py [following pep, removed extra blank lines]

### CODE REVIEW

File: webapp/coderplex_apis/settings.py [hunk 1]
```diff
@@ -79,8 +79,12 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'coderplex-backend',
+        'USER': 'postgres',
+        'PASSWORD': 'c$@Ode12859444',
```
Comment (vinaypuppal) [inline webapp/coderplex_apis/settings.py]: Don't commit `user` and `password` publicly. 
> May be store them in system env variables and access them at runtime.

@rrmerugu I don't how how you achieve this in python but it is not good to commit database credentials in git?

Comment (rrmerugu) [reply webapp/coderplex_apis/settings.py]: @djsbalakrishnan  Always use something like  `os.environ.get('username')` for the configurations

File: webapp/member/models.py [hunk 2]
```diff
@@ -5,6 +5,10 @@
 
 # Create your models here.
 class UserProfile(models.Model):
+    """
```
Comment (rrmerugu) [inline webapp/member/models.py]: Why do you have this comments in this PR ?  Probably next time keep the PRs strictly to the concerned tasks. If the PR is not merged, your corrections are waste of your time.

File: requirements/requirements.txt [hunk 3]
```diff
@@ -1,3 +1,6 @@
 Django==1.11.6
 djangorestframework==3.7.1
-Pillow==4.3.0
\ No newline at end of file
+olefile==0.44
+Pillow==4.3.0
```
Comment (rrmerugu) [inline requirements/requirements.txt]: are we using any imagefield in this project ? why is this pillow package here ?

File: webapp/coderplex_apis/settings.py (hunk 1 above)
Comment (djsbalakrishnan) [reply webapp/coderplex_apis/settings.py]: @vinaypuppal @rrmerugu I have changed the configuration. I have committed the changes

File: requirements/requirements.txt (hunk 3 above)
Comment (djsbalakrishnan) [reply requirements/requirements.txt]: It was there when I forked this project. Do you want me to uninstall this module and push ?

File: webapp/member/models.py (hunk 2 above)
Comment (djsbalakrishnan) [reply webapp/member/models.py]: Okay will make sure that I stick to the task.

**Comment by djsbalakrishnan** [comment]
I have modified the fields in the UserProfile model to accept null values. Please check

**Comment by djsbalakrishnan** [comment]
I have pushed the updated code which is not working, please look into files changed and update this pull request.

**Pull request merged**